src/layers/VivViewerLayer/VivViewerLayer.js

```diff
@@ -63,13 +66,28 @@ export default class VivViewerLayer extends CompositeLayer {
       dtype,
       colorValues: paddedColorValues,
       sliderValues: paddedSliderValues,
+      // We want a no-overlap caching strategy with an opacity < 1 to prevent
+      // multiple rendered sublayers (some of which have been cached) from overlapping
+      refinementStrategy: opacity === 1 ? 'best-available' : 'no-overlap',
       // TileLayer checks `changeFlags.updateTriggersChanged.getTileData` to see if tile cache
       // needs to be re-created. We want to trigger this behavior if the loader changes.
       // https://github.com/uber/deck.gl/blob/3f67ea6dfd09a4d74122f93903cb6b819dd88d52/modules/geo-layers/src/tile-layer/tile-layer.js#L50
       updateTriggers: {
         getTileData: [loader]
       }
     });
+    // This gives us a background image and also solves the current
+    // minZoom funny business.  We don't use it for the background if we have an opacity
+    // paramteter set to anything but 1, but we always use it for situations where
+    // we are zoomed out too far.
+    const baseLayer = new StaticImageLayer(this.props, {
+      id: `StaticImageLayer-${loader.type}`,
+      scale: 2 ** (-minZoom - 1),
+      imageHeight: tileSize,
+      imageWidth: tileSize,
+      visible: opacity === 1 || minZoom > this.context.viewport.zoom
+    });
+    const layers = [baseLayer, tiledLayer];
```

Comment:
Thanks for adding a comment here. I think this is a good strategy and can't think of anything else (unless deck.gl decides to implement something).